uvicorn/config.py

```diff
@@ -360,12 +374,16 @@ def load(self):
 
         self.loaded = True
 
-    def setup_event_loop(self):
-        loop_setup = import_from_string(LOOP_SETUPS[self.loop])
-        if loop_setup is not None:
-            loop_setup()
+    def setup_event_loop(self) -> None:
+        loop_setup_str: Optional[str] = LOOP_SETUPS[self.loop]
+        if loop_setup_str:
+            loop_setup: Callable = import_from_string(loop_setup_str)
+            if not inspect.isfunction(loop_setup):
+                raise ImportFromStringError("Asyncio event loop must be a callable.")
```

Comment:
@euri10 take special attention to this, plz
I'm not sure...

Reply:
Yeah, I wasn't sure of the best approach here either. Open to suggestions.

Reply:
I would type ignore that part